src/Main.java

```diff
@@ -1,5 +1,70 @@
+import java.util.Arrays;
+import java.util.Locale;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        task1();
+        task2();
+        task3();
+        task4();
     }
-}
\ No newline at end of file
+    public static int[] generateRandomArray() {
+        java.util.Random random = new java.util.Random();
+        int[] arr = new int[30];
+        for (int i = 0; i < arr.length; i++) {
+            arr[i] = random.nextInt(100_000) + 100_000;
+        }
+        return arr;
+    }
+    public static void task1() {
+        int[] arr = generateRandomArray();
+        System.out.println("\nЗадача 1");
+        int sum = 0;
+        for (int expense : arr) {
+            sum += expense;
+        }
+        System.out.println("Сумма трат за месяц составила " + sum + " рублей");
+    }
+    public static void task2() {
+        int[] arr = generateRandomArray();
+        System.out.println("\nЗадача 2");
+        //сначала вычисляем максимальное значение элемента массива
+        int maxExpense = 0;
+        for (int currentExpense : arr) {
+            if (currentExpense > maxExpense) {
+                maxExpense = currentExpense;
+            }
+        }
+        //затем вычисляем минимальное значение
+        int minExpense = maxExpense;//инициализируем переменную - задаём уже известное максимальное значение
+        for (int currentExpense : arr) {
+            if (currentExpense < minExpense) {
+                minExpense = currentExpense;
+            }
+        }
+        System.out.println("Минимальная сумма трат за день составила " + minExpense + " рублей");
+        System.out.println("Максимальная сумма трат за день составила " + maxExpense + " рублей");
+        //в этой задаче можно было обойтись одним циклом for each, но тогда надо было бы использовать
+        //верхний лимит выплат из условий задачи (200_000 р) для начального значеня переменной minExpense.
+        //я хотел обойтись без магических чисел и поэтому сделал задачу в 2 этапа. возможно, существует и другое решене...
+
+    }
+    public static void task3() {
+        int[] arr = generateRandomArray();
+        System.out.println("\nЗадача 3");
+        double days = 30;
```

Comment:
почему не 100? дан массив, значит надо использовать длину массива, а не брать числа с потолка
```suggestion
        double days = arr.length;
```

Reply:
Дима, число не с потолка, а из условий задачи:  Вот, смотри: Задача 3
Теперь бухгалтерия хочет понять, какую в среднем сумму компания тратила в течение 30 дней.

Reply:
Да, но 30 дней у тебя - это длина массива, потому что в каждой ячейке - значение для дня. Если массив будет длины 31, то у тебя ошибка будет в программе - потому что забудешь поменять 30 на 31. И так каждый раз. Поэтому надо использовать длину массива, а не число

Reply:
Дима, понял, спасибо!